tests/playwright/sanity/basic-gallery.test.ts

```diff
@@ -60,6 +62,34 @@ test( 'Basic Gallery Lightbox test with older Swiper', async ( { page, apiReques
 	await testBasicSwiperGallery( editor );
 } );
 
+test.only( 'Basic Gallery Lightbox test with breakpoints', async ( { page, apiRequests }, testInfo ) => {
+	// Arrange.
+	const wpAdmin = new WpAdminPage( page, testInfo, apiRequests );
+	const editor = await wpAdmin.openNewPage();
+	const breakpoints = new Breakpoints( page );
+
+	await editor.closeNavigatorIfOpen();
+
+	await editor.addWidget( 'image-gallery' );
+	await editor.openPanelTab( 'content' );
+	await editor.addImagesToGalleryControl();
+	await editor.setSelectControlValue( 'open_lightbox', 'yes' );
+
+	await editor.publishPage();
+
+	await breakpoints.setBreakpoint( editor, 'mobile', viewportSize.mobile.width );
+
+	// Act.
+	const pageId = await editor.getPageId();
```

Comment:
Maybe extract to `viewPage()`, as we have in the Pro repo: 

https://github.com/elementor/elementor-pro/blob/a526f2337aa09480b47a75f37a87493750f6c437/tests/playwright/pages/editor-page.ts#L501